src/bindings/python/src/openvino/runtime/opset12/__init__.py

```diff
@@ -120,7 +120,7 @@
 from openvino.runtime.opset1.ops import prior_box_clustered
 from openvino.runtime.opset1.ops import psroi_pooling
 from openvino.runtime.opset4.ops import proposal
-from openvino.runtime.opset1.ops import range
+from openvino.runtime.opset12.ops import range
```

Comment:
If `def range` will be moved to `opset4/ops.py`:
```suggestion
from openvino.runtime.opset4.ops import range
```

Reply:
Done